src/components/Form/index.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Dispatch,
+  FC,
+  SetStateAction,
+  useState,
+} from 'react';
+import usersFromServer from '../../api/users';
+import { TodoItem } from '../../types/todo';
+
+const Form: FC<{ updateState: Dispatch<SetStateAction<TodoItem[]>> }> = ({
```

Comment:
better to use interface for type  Prop and why do you need to use so difficult type for set func
https://bobbyhadz.com/blog/react-typescript-pass-function-as-prop